compiler-rt/lib/tsan/rtl/tsan_platform_linux.cpp

```diff
@@ -214,6 +214,88 @@ void InitializeShadowMemoryPlatform() {
 
 #endif  // #if !SANITIZER_GO
 
+#  if !SANITIZER_GO
+static void ReExecIfNeeded() {
+  // Go maps shadow memory lazily and works fine with limited address space.
+  // Unlimited stack is not a problem as well, because the executable
+  // is not compiled with -pie.
+  {
```

Comment:
what is the purpose of whole function {}  ?

Reply:
I guess it's copy-pasted
it's noop here, please remove.

Reply:
Done!